features/FieldTypeForm/selection_fieldtype_edit_form.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Selection field value edit form
+    In order to edit content of a Selection field
+    As an integrator
+    I want the Selection field form to implement the FieldType's behaviour
+
+Background:
+    Given a Content Type with an ezselection field definition
+
+Scenario: The attributes of the field have a form representation
+    Given I view the edit form for this field
+     Then the edit form should contain a fieldset named after the field definition
+      And it should contain a select field
+
+Scenario: The options added to a field definition have a form representation
+    Given I add the following options to the field definition:
+          | one |
+          | two |
+          | three |
+     When I view the edit form for this field
+     Then the following options should be represented in the form:
```

Comment:
By dropdown, do you mean the JS widget in PlatformUI? The BDD of repoforms itself uses plain html, doesn't it? So then we may need to cover both dropdown and multiselect cases.
2nd complication: In the current PlatformUI, the items in the select are NOT in fact represented in the form, not until you start typing anyway. Can we make BDD sentences that make sense both in plain html and in PlatformUI?